src/Assets/svgs/ArrowRotation.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+export default function ArrowRotation() {
+  return (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <g id="arrow-rotation">
+        <path
+          id="Vector 30"
+          d="M18.5355 5.53553L18.9077 12.2344M18.5355 5.53553L11.8366 5.16337M18.5355 5.53553L6.51472 17.5563"
+          stroke="#8C8C8C"
+          stroke-width="2"
+          stroke-linecap="round"
+          stroke-linejoin="round"
```

Comment:
`strokeWIdth="2"` `strokeLinecap="round"` `strokeLinejoin="round"`
로 수정해야 콘솔에 svg오류가 안날 것 같아요!

```suggestion
          strokeWidth="2"
          strokeLinecap="round"
          strokeLinejoin="round"
```